### DISCUSSION TITLE
Not able to create login user. Receiving error. Please help me to resolve this

### DISCUSSION THREAD

**Discussion by hiralalmahival007** [postalserver/postal#2706] [Help with using Postal]
Not able to create login user. Receiving error. Please help me to resolve this
![Screenshot from 2023-11-24 16-37-58](https://github.com/postalserver/postal/assets/142792427/55d3e919-1c30-48ee-8b07-b57ced5561b8)

**Comment by willpower232** [comment]
Have a look at #2508

**Reply by hiralalmahival007** [reply]
It worked! Thanks a lot!
